download-data.sh

```diff
@@ -21,6 +21,14 @@ do
   curl --create-dirs $URL/$RELEASE/$file -o data/$RELEASE/$file -z data/$RELEASE/$file
 done
 
+# Download reference and gencode file from public ftp
+REFERENCE="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/GRCh38.primary_assembly.genome.fa.gz"
+GENCODE="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/gencode.v27.primary_assembly.annotation.gtf.gz"
+cd data/$RELEASE
```

Comment:
We don't expect these files to change and we probably want them in data so
```suggestion
cd data
```
seems more appropriate to me.